docs/streams/developer-guide/write-streams.html

```diff
@@ -78,26 +78,36 @@
                   <td><code class="docutils literal"><span class="pre">{{fullDotVersion}}</span></code></td>
                   <td>(Required) Kafka client library.  Contains built-in serializers/deserializers.</td>
               </tr>
+              <tr class="row-even"><td><code class="docutils literal"><span class="pre">org.apache.kafka</span></code></td>
+                  <td><code class="docutils literal"><span class="pre">kafka-streams-scala</span></code></td>
+                  <td><code class="docutils literal"><span class="pre">{{fullDotVersion}}</span></code></td>
+                  <td>(Optional) Scala DSL API library to write Scala Kafka Streams applications.</td>
```

Comment:
Scala version missing?

Reply:
I think the Scala version cannot be changed when specifying the `kafka-streams-scala` artifact, as it is encapsulated when that artifact is compiled already. @debasishg please correct me if I'm wrong.

Reply:
Not sure what you mean @guozhangwang. The Scala version is usually part of the artifact name.

Reply:
:+1: